data/sql_updates/create_committee_history.sql

```diff
@@ -79,35 +67,33 @@ select distinct on (dcp.cmte_sk, cycle)
     dcp.cmte_custodian_title as custodian_name_title,
     dcp.cmte_custodian_zip as custodian_zip,
     -- Properties
-    dcp.cmte_email as email,
+    fec_yr.cmte_email as email,
     dcp.cmte_fax as fax,
-    dcp.cmte_web_url as website,
+    fec_yr.cmte_url as website,
     dcp.form_tp as form_type,
     dcp.leadership_pac as leadership_pac,
-    dcp.load_date as load_date,
```

Comment:
Context? Why do you have to get rid of load_date and what are you using 
the date to accomplish?



From:   Lindsay Young <notifications@github.com>
To:     18F/openFEC <openFEC@noreply.github.com>
Cc:     jwchumley <jchumley@fec.gov>
Date:   11/06/2015 12:32 PM
Subject:        Re: [openFEC] Feature/update committee data (#1325)



In data/sql_updates/create_committee_history.sql:
>      dcp.form_tp as form_type,
>      dcp.leadership_pac as leadership_pac,
> -    dcp.load_date as load_date,

If we have to get rid of load_date from dimcmteproperties, we can add 
date_entered and date_changed.
@jwchumley and @paulclark2 do you think that date_entered is comparable 
enough to load_date that we could call it that?
?
Reply to this email directly or view it on GitHub.